### ISSUE TITLE
Server crash when redstone signal applied to redstone interface

### ISSUE THREAD

**Issue by ghost** [enhancedportals/enhancedportals#12]
I have a redstone interface on my portal set to Dial Specific Identifier (2) and a Dialing Device with 3 saved portals (all three work when opened manually). When I apply a redstone signal, the server locks up until the signal stops, but no portal is ever opened. Using a lever and a line of redstone dust locks the server up entirely. Here's the error message that gets thrown at least 1x per tick while the signal is active.

http://pastebin.com/xNFp5qYu

I'm on 3.0.12 using Forge 10.13.4.1517 as part of the Space Astronomy modpack version 1.3.4.

**Comment by MJRLegends** [comment]
Yeah seems to be a mod problem so we will wait to see what they come back with.

**Comment by daydriem** [comment]
I am also having this bug.
I use the following setup:
Three portals, each have three redstone interfaces. One redstone interface with create portal on pulse and an attached button. The second redstone interface has input destroy on signal and the third is connected to it and has output entity touches portal.
It seems to work as long as I have two portals, as soon as I add a third, it breaks. Users report it sometimes occurring when pressing the button to open the portal, other times when using the portal.

http://pastebin.com/wbaTnVyR

The bug was occurring on 3.0.10, so I upgraded to 3.0.12, unfortunately it is still occurring. I am running Forge 1.13.2.1291. I hope that helps. If there is anything else I can provide, please let me know.
